nntrainer/layers/conv2d_layer.cpp

```diff
@@ -26,30 +26,71 @@
 #include <util_func.h>
 
 namespace nntrainer {
-
-/// @note this will be deleted after conv2d optimization is done!
-#ifdef PROFILE
 namespace {
 
-int pad_profile_key;
-int conv_gemm_profile_key;
-int im2col_key;
-int add_bias_key;
-int clean_up;
-int temp_key;
-
-void register_event() {
-  pad_profile_key = profile::Profiler::Global().registerEvent("zero_pad");
-  im2col_key = profile::Profiler::Global().registerEvent("im2col");
-  conv_gemm_profile_key =
-    profile::Profiler::Global().registerEvent("conv_gemm");
-
-  add_bias_key = profile::Profiler::Global().registerEvent("add_bias_key");
-  clean_up = profile::Profiler::Global().registerEvent("clean_up");
-  temp_key = profile::Profiler::Global().registerEvent("temp_key");
+void col2im(const Tensor &col_matrix, const TensorDim &kdim,
```

Comment:
this should be static.
Also, this is conv2d class static member.

Reply:
I think there is no reason for im2col and col2im be exposed outside of this cpp except for the reason stated above.

So it is hidden inside `conv2d_layer.cpp` with anonymous namespace

Reply:
then, for now, let's just make it a static function in this cpp.

Reply:
Done